lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -121,7 +120,7 @@ def test_as_staff_for_honor(self):
         self.verify_response(params={'username': self.honor_user.username})
 
     def test_as_anonymous_user(self):
-        self.verify_response(expected_status_code=401)
+        self.verify_response(expected_status_code=200)
```

Comment:
The 401 was being caused by the removed class decorator on  lms.djangoapps.course_api.views.CourseDetailView.  That wasn't supposed to be there.